pom.xml

```diff
@@ -376,6 +377,28 @@
         </executions>
       </plugin>
 
+      <!-- Enforce versions -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>1.4</version>
+        <executions>
+          <execution>
+            <id>enforce-versions</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireJavaVersion>
```

Comment:
Could it just be one section that looks like this? Then if we upgrade the maven-enforcer-plugin we only need to change on place.
```
<rules>
  <!-- Enforce Hadoop version -->
  <requireProperty>
    <property>hadoop.version</property>
    <regex>[^12]\..*|2\.[4-9]\..*</regex>
    <regexMessage>
      hadoop.version must be 2.4.0 or later to build yarn integration.
    </regexMessage>
  </requireProperty>

  <!-- Enforce Java version -->
  <requireJavaVersion>
    <version>${java.version}</version>
  </requireJavaVersion>
</rules>```